roles/core/nfs_client/readme.rst

```diff
@@ -4,21 +4,53 @@ NFS client
 Description
 ^^^^^^^^^^^
 
-This role provides an automatic nfs client that mount targets and create local directories if needed.
+This role provides an automatic nfs client that mount targets and create local
+directories if needed.
 
 Instructions
 ^^^^^^^^^^^^
 
 All configuration is done in *group_vars/all/general_settings/nfs.yml*.
 
-See the **nfs_server** role isntructions for more details.
+See the **nfs_server** role instructions for more details.
 
-Simply note that path that should be mounted will be automatically created by this role if they do not exist.
+Simply note that path that should be mounted will be automatically created by
+this role if they do not exist.
+
+Input
+^^^^^
+
+Mandatory inventory vars:
+
+**hostvars[inventory_hostname]**
+
+* nfs[item]
+  * .mount
```

Comment:
The RST rendering in GitHub is weird: https://github.com/bluebanquise/bluebanquise/blob/16308753e084a3920c8aa3666a5ebe053d3dcaee/roles/core/nfs_client/readme.rst

Are you sure about the syntax?

Reply:
Missing one space.